apps/api/models.py

```diff
@@ -102,24 +105,50 @@ class APIKey(models.Model):
     class Meta:
         db_table = 'api_key'
 
+    def __unicode__(self):
+        return (
+            u'<{cls} user={user}, type={type}, key={key} secret=...>'
+            .format(cls=self.__class__.__name__, key=self.key,
+                    type=self.type, user=self.user))
+
     @classmethod
     def get_jwt_key(cls, **query):
```

Comment:
I'm just thinking if we end up with rows that have nulls or empty strings intentionally or unintentionally someone might be able to pass in an empty user or key and get that row returned. I guess they'd need to know the corresponding secret to do anything though so that probably doesn't matter.